### PR TITLE
meal-sharing hw

### PR DESCRIPTION
**Pull request by Kositthai**
Meal-sharing homework

### CODE REVIEW

File: src/backend/app.js [hunk 1]
```diff
@@ -21,10 +23,53 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+const db = Knex({
```
Comment (Divyajg) [inline src/backend/app.js]: The database should be connected through a .env file as this info contains sensitive information like username and password. This is the crucial thing that a developer should be aware of.

**Review by Divyajg — CHANGES_REQUESTED** [review]
Reconnect your database to the backend using .env file.

File: src/backend/app.js (hunk 1 above)
Comment (Kositthai) [reply src/backend/app.js]: Hi @Divyajg  Thank you so much for your feedback.
I have fixed the code to connect through .env now :)

**Review by Divyajg — COMMENTED** [review]
1. While calling an api, if something went wrong, there will be an error and we should handle that error to convey it to the user. To handle errors: we can use try/catch or if/else
2. If you found no meals, it is not an error.

**Comment by Kositthai** [comment]
> 1. While calling an api, if something went wrong, there will be an error and we should handle that error to convey it to the user. To handle errors: we can use try/catch or if/else
> 2. If you found no meals, it is not an error.

@Divyajg  I have added try and catch in my code now, Kindlly check again :)

**Review by Divyajg — APPROVED** [review]
Good work. keep coding.

**Pull request merged**